voir/instruments/gpu/rocm.py

```diff
@@ -205,7 +205,6 @@ def get_gpu_info(self, device):
             "temperature": temp,
             "power": power,
             "selection_variable": "ROCR_VISIBLE_DEVICES",
-            "driver": rsmi.smi_get_kernel_version(),
```

Comment:
This is not the right call, but https://github.com/breuleux/voir/pull/33 fixes this anyway